lib/options.py

```diff
@@ -75,6 +75,14 @@ def __init__(self):
                 help="""Print TAP13 test output to log.
                 Default: false.""")
 
+        parser.add_argument(
+                "--debug",
+                dest='debug',
```

Comment:
Why single quotes?

Reply:
I generally prefer single quotes in Python and Lua (without any reasons, just habit). I have to be consistent with the surrounding code, but there is no consistency within the file. I'll use single quotes within the code block so (if there are no objections).

Reply:
Done.